### DISCUSSION TITLE
Suggestion: Move detection/alert acknowledge all button to the top of the page 0.14

### DISCUSSION THREAD

**Discussion by superdong69** [blakeblackshear/frigate#11608] [Ideas]
I get a ton of detections that aren't relevant in any way, I'd like to be able to clear them easily, or even turn them off entirely if possible. 
I'm sure there are people who also just care about alerts. For me, these detections are essentially false positives. 
Also, there doesn't appear to be a way to get to frigate plus like there used to be. Once I click "not a person" the event is gone where before I could click, go to frigate +. 

Finally, on the frigate + tab, there's no clear way to select all and delete. If there is, I'd love to know, but also think it should be a little more obvious. It's not "necessary" but I'd imagine it would fill up the hard drive without pruning?

**Comment by NickM-27** [comment]
You should just configure your detections to not save what you don't care about. 

The button is at the bottom because you press it after you've looked over all the detections.

**Reply by superdong69** [reply]
I apologize. I don't know every little intricacy of the application. Looking at my detections they are all cars outside of my zones, and I'm not sure where I would go to fine tune that.

**Reply by hawkeye217** [reply]
Take a look at the review docs: https://deploy-preview-11419--frigate-docs.netlify.app/configuration/review

You can exclude items from alerts and detections or have them only for specific zones:

https://deploy-preview-11419--frigate-docs.netlify.app/configuration/zones#restricting-alerts-and-detections-to-specific-zones

**Reply by NickM-27** [reply]
See https://deploy-preview-11419--frigate-docs.netlify.app/configuration/review#restricting-detections-to-specific-labels

I similarly removed cars from detections so I just see animals and people that are outside of zones

**Reply by superdong69** [reply]
AMAZING STUFF!!!!! Thank You guys! What about the other idea though about making Frigate + a bit easier to deal with?

**Reply by NickM-27** [reply]
there is no delete function, this page just follows your already existing snapshots retention

**Reply by superdong69** [reply]
Oh ok. That's good to know then. So I can just ignore it. Nice!

**Comment by OnFreund** [comment]
While I think the button makes sense at the bottom, I will say that in many cases I prefer reviewing from earliest to latest, so being able to order in reverse would help (rather than starting at the bottom, going up, and then scrolling down again for the button).
